recipes/nvmolkit/recipe.yaml

```
@@ -0,0 +1,66 @@
+context:
+  version: "0.2.0"
+
+package:
+  name: "nvmolkit"
+  version: ${{ version }}
+
+source:
+  url: https://github.com/NVIDIA-Digital-Bio/nvMolKit/archive/v${{ version }}.tar.gz
+  sha256: 8c0181c4dcd02aac42dc3c1b783ac677a8149fd5d5b5d25e85ba30e85bd6ec5a
+
+build:
+  number: 0
+  skip:
+    - osx
+    - win
```

```suggestion
  skip:
    - osx
    - win
    - cuda_compiler = "None"
    - py<311
```
I think this should skip the non-CUDA build on Linux and the Python versions below 3.11 - let's see if that works.